### ISSUE TITLE
Production site not running minified build of React

### ISSUE THREAD

**Issue by davidrf** [tidepool-org/blip#423]
Hello! Awesome site by the way! While browsing, I noticed the following React dev tools warning:

![screen shot 2017-06-01 at 10 02 53 am](https://cloud.githubusercontent.com/assets/9678182/26683349/84489224-46b1-11e7-87cc-188639db8612.png)

I'm not intimately familiar with your build process, but this may or may not be fixed by adding a `-p` flag to [this line](https://github.com/tidepool-org/blip/blob/master/buildapp.js#L15), so it becomes:

```js
exec('webpack --entry \'' + entry + '\' --output-filename \'bundle.[hash].js\' --devtool source-map --colors --progress -p');
```

**Comment by cbwebdevelopment** [comment]
Thanks for the feedback @davidrf!

We will be including this in an upcoming build.

**Comment by cbwebdevelopment** [comment]
Fixed in #427
